src/sentry/migrations/0637_backfill_priority_for_groups.py

```diff
@@ -0,0 +1,121 @@
+# Generated by Django 3.2.23 on 2024-01-23 23:32
+
+import logging
+
+from django.db import connection, migrations
+from psycopg2.extras import execute_values
+
+from sentry.issues.grouptype import (
+    GroupCategory,
+    PerformanceP95EndpointRegressionGroupType,
+    ProfileFunctionRegressionType,
+    get_group_type_by_type_id,
```

Comment:
I guess this might not be possible to copy in though...

Reply:
I could copy just the type IDs? like `PERFORMANCE_P95_ENDPOINT_REGRESSION_GROUPTYPE_ID = 1018` and use that instead of getting it from `PerformanceP95EndpointRegressionGroupType`.